mteb/evaluation/evaluators/ClassificationEvaluator.py

```diff
@@ -14,6 +19,66 @@
 logger = logging.getLogger(__name__)
 
 
+def dot_distance(a: np.ndarray, b: np.ndarray) -> float:
+    return -np.dot(a, b)
+
+
+class kNNMultiLabelClassificationEvaluator(Evaluator):
+    def __init__(
+        self,
+        embeddings_train,
+        y_train,
+        embeddings_test,
+        y_test,
+        k=1,
+        batch_size=32,
+        limit=None,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        if limit is not None:
```

Comment:
```suggestion
        if limit:
```

Reply:
I frankly don't think this is a good idea, staying explicit about this is better in my opinion